core/chains/evm/log/broadcaster.go

```diff
@@ -494,12 +495,11 @@ func (b *broadcaster) onReplayRequest(replayReq replayRequest) {
 	b.backfillBlockNumber.Int64 = replayReq.fromBlock
 	b.backfillBlockNumber.Valid = true
 	if replayReq.forceBroadcast {
-		ctx, cancel := b.chStop.NewCtx()
+		ctx, cancel := b.chStop.CtxCancel(context.WithTimeout(context.Background(), time.Minute))
```

Comment:
This was limited to just the one call to `MarkBroadcastsUnconsumed` before. I think we need to preserve that.

Reply:
Would wrapping this in a `func() {}()` be sufficient so the ctx is canceled after?

Reply:
Oh I misread the collapsed github display. If this is only passed to `MarkBroadcastsUnconsumed` then it is fine as is.